frontend/lib/ui/Button/Button.module.css

```diff
@@ -78,6 +78,40 @@
     }
   }
 
+  &.tertiary {
+    --button-border-width: 0px;
+    border: none;
+    background: transparent;
+    color: var(--color-error-darkest);
+
+    &:not(:hover, :active) {
+      box-shadow: none;
+    }
+
+    &:hover {
+      background: #fef9fa;
+    }
+
+    &:focus,
+    &:focus-visible {
+      box-shadow: none;
+      outline: none;
+      --button-border-width: 2px;
```

Comment:
what does this do?

Reply:
This CSS var is used to accommodate for different border sizes for different button states, without making the button size change. This was done in https://github.com/kiesraad/abacus/pull/591/files